doc/openapi/v0/cyphernode-api.yaml

```diff
@@ -1822,6 +1822,55 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ApiResponseTemporarilyUnavailable'
+  /bitcoin_generatetoaddress:
+    post:
+      tags:
+        - "core features"
+        - "bitcoin"
+      summary: "Mine blocks immediately to a specified address (before the call returns)"
+      description: "Mine blocks immediately to a specified address (before the call returns)"
+      operationId: "generateToAddress"
+      requestBody:
+        description: "Number of blocks, destination address, max number of iterations to try"
+        required: true
+        content:
+          application/json:
+            schema:
+              type: "object"
+              properties:
+                nbBlocks:
```

Comment:
Make the arguments optional and the request HTTP-GET compatible

Reply:
Done!